### PR TITLE
[WIP] Release candidate

### PR DESCRIPTION
**Pull request by ianegordon**
Do NOT Merge.

### CODE REVIEW

File: components/Dialogs/README.md [hunk 1]
```diff
@@ -187,6 +188,49 @@ MDCAlertAction *alertAction =
 
 ## Extensions
 
+<!-- Extracted from docs/theming.md -->
+
+### Theming
+
+You can theme an MDCDialog to match the Material Design Dialog using your app's schemes in the DialogThemer
```
Comment (galiak11) [inline components/Dialogs/README.md:195]: an MDCAlertController. No such thing as MDCDialog.

File: components/Dialogs/README.md [hunk 2]
```diff
@@ -187,6 +188,49 @@ MDCAlertAction *alertAction =
...
+You can then import the extension and create an `MDCAlertControllerThemer` instance. A dialog scheme defines
+the design parameters that you can use to theme your dialogs.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+// Step 1: Import the DialogThemer extension
+import MaterialComponents.MaterialDialogs_DialogThemer
+
+// Step 2: Create or get a alert scheme
```
Comment (galiak11) [inline components/Dialogs/README.md:213]: tip: "an alert" (missing "n").
(and same for the obj-c step 2 below).

**Review by galiak11 — APPROVED** [review]

**Comment by ianegordon** [comment]
All committers are Googlers or have signed the CLA.

File: components/Dialogs/README.md (hunk 1 above)
Comment (ianegordon) [reply components/Dialogs/README.md:195]: Any changes should be made upstream (in the develop branch).

File: components/Dialogs/README.md (hunk 2 above)
Comment (ianegordon) [reply components/Dialogs/README.md:213]: Any changes should be made upstream (in the develop branch).

**Pull request merged**